### PR TITLE
onKeyPress is replaced with onKeyDown

### PR DESCRIPTION
**Pull request by themukeshmahiya**
trying to solve this good first issue 

https://github.com/cockpit-project/cockpit/issues/19061

### CODE REVIEW

File: pkg/lib/cockpit-components-logs-panel.jsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export class JournalOutput {
         }
 
         // only consider enter button for keyboard events
-        if (ev.type === 'keypress' && ev.key !== "Enter")
+        if (ev.type === 'KeyDown' && ev.key !== "Enter")
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-logs-panel.jsx:58]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-logs-panel.jsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ export class JournalOutput {
             <div className="cockpit-logline" role="row" tabIndex="0" key={entry.__CURSOR}
                 data-cursor={entry.__CURSOR}
                 onClick={ev => this.onEvent(ev, entry.__CURSOR, full_content)}
-                onKeyPress={ev => this.onEvent(ev, entry.__CURSOR, full_content)}>
+                onKeyDown={ev => this.onEvent(ev, entry.__CURSOR, full_content)}>
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-logs-panel.jsx:81]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/side-panel.jsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ export class SidePanel extends React.Component {
             show_all_button = (
                 <FlexItem alignSelf={{ default: 'alignSelfCenter' }}>
                     <Button variant='link'
-                            onKeyPress={ev => ev.key === "Enter" && this.setState({ collapsed: false })}
+                            onKeyDown={ev => ev.key === "Enter" && this.setState({ collapsed: false })}
```
Comment (cockpituous) [inline pkg/storaged/side-panel.jsx:63]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/side-panel.jsx [hunk 4]
```diff
@@ -113,7 +113,7 @@ class SidePanelRow extends React.Component {
                 return;
 
             // only consider enter button for keyboard events
-            if (event.type === 'keypress' && event.key !== "Enter")
+            if (event.type === 'KeyDown' && event.key !== "Enter")
```
Comment (cockpituous) [inline pkg/storaged/side-panel.jsx:116]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/storage-controls.jsx [hunk 5]
```diff
@@ -78,7 +78,7 @@ function checked(callback) {
             return;
 
         // only consider enter button for keyboard events
-        if (event.type === 'keypress' && event.key !== "Enter")
+        if (event.type === 'KeyDown' && event.key !== "Enter")
```
Comment (cockpituous) [inline pkg/storaged/storage-controls.jsx:81]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by martinpitt — APPROVED** [review]
Thank you!

**Pull request merged**